build.sh

```diff
@@ -0,0 +1,55 @@
+######################################################################
+# Copyright (c) 2019 Claudio André <claudioandre.br at gmail.com>
+#
+# This program comes with ABSOLUTELY NO WARRANTY; express or implied.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, as expressed in version 2, seen at
+# http://www.gnu.org/licenses/gpl-2.0.html
+######################################################################
+
```

Comment:
This should be upstream, not sure if it's something flatpak specific or you can not achieve with a "real" build system.

Reply:
This is packaging specific. A real user builds the binary he/she is going to use. A package should have SSE2, AVX, .... And them (the binaries packaged) have to fallback.